### DISCUSSION TITLE
Smart embeds

### DISCUSSION THREAD

**Discussion by jmduke** [buttondown/discussions#3] [Ideas]
- [x] TikTok
- [ ] audio.com
- [ ] Wistia
- [ ] Simplecast
- [ ] Buttondown
- [ ] Calendly

**Comment by jmduke** [comment]
Just added support for **Soundcloud**!

**Comment by Natha** [comment]
Hey Justin !

Since you asked if we had other ideas for markdown integration, here's my list (in order of priority for us) : 

[Spotify](https://developer.spotify.com/documentation/embeds/) player
[Apple Music](https://tools.applemediaservices.com/album/1667328437?country=be) player
[Tidal](https://embed.tidal.com/#/) player
[Songkick](https://www.songkick.com/) tour list
[Bandsintown](https://www.artist.bandsintown.com/widget) tour list

Many thanks for your awesome work :)

**Reply by jmduke** [reply]
Spotify going live right now, just for you :)

![image](https://user-images.githubusercontent.com/461046/230172037-d0b23333-aaeb-43dd-936a-a55b3f559b4e.png)

![image](https://user-images.githubusercontent.com/461046/230172155-9dfb0f54-b0fd-4b4b-b764-10800bbd0c71.png)

**Reply by Natha** [reply]
Damn Justin ! **That** is what I call customer support. Thank you so much !! (By the way, feel free to subscribe to our [newsletter](https://buttondown.email/tempestofnoise) 👀)

**Comment by fclough** [comment]
A Quizlet embed would be appreciated....

**Reply by jmduke** [reply]
This is a new one! What kind of use case do you have in mind?

**Comment by fclough** [comment]
I just started to produce a monthly newsletter for a website I run for some friends who teach shorthand. Quizlet is used to create the study packs of shorthand outlines and I would like to include an embed in the newsletter to highlight the new adds over the prior month.   
  

  
  
  
  
  
  
  

  
  
>

**Reply by jmduke** [reply]
Awesome — thanks for the elaboration. Could you share an example study pack URL? (Feel free to email me if you don't want it to be public!)

**Comment by fclough** [comment]
This should work
  

  
 https://quizlet.com/752166970/lltt-community-takes-steps-to-help-those-suffering-hardship-flash-cards/?i=4litj&x=1jqY
  
  
  
  
  
  
  

  
  
>

**Reply by jmduke** [reply]
Excellent, thank you!

**Comment by jmduke** [comment]
Aaand we just launched Transistor:

https://buttondown.email/changelog/2024-04-12

**Comment by krangor** [comment]
Dare I suggest... Buttondown?

Substack has a nice feature in which posting a link to a Substack page creates a nice little card preview for it. Would be really cool to have something like that here.

**Reply by jmduke** [reply]
Ah, I love that!

**Comment by JayCuthrell** [comment]
Just dropping in to say the recent updates to unfurl / oembed / etc... are wonderful. Thank you!

**Reply by jmduke** [reply]
I believe the credit goes to the inimitable @tk04 for that :)

**Comment by JayCuthrell** [comment]
Would it be possible to limit the unfurl / oembed image size to a percentile vs. full width throughout a post or as a general setting?

Context: My newsletter contains many many Techmeme.com URLs https://hot.fudge.org/archive/hot-fudge-sunday-for-2025-03-30/

**Reply by jmduke** [reply]
Hmm, can you talk more about what you're looking to achieve? I'm not sure I follow.

**Reply by JayCuthrell** [reply]
I include Techmeme URLs (often) and those work 99% of the time. I'd like to be able to set the image embedded sizing to a percentile or other maximum width.

<img width="600" alt="Screenshot 2025-04-13 at 1 08 58 PM" src="https://github.com/user-attachments/assets/b1c55d09-a7a3-4d0a-a34b-73d58fb6ee8b" />


I'd also like to see a way to determine why a URL I include for embedding isn't working. It is hit or miss sometimes in Preview. However, some URLs that can be validated in Linkedin Post Inspector do not embed or display rich media.

Example: 

- URL: https://www.nexustek.com/blog/the-hidden-roi-of-strategic-it-cost-management
- Test: https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fwww.nexustek.com%2Fblog%2Fthe-hidden-roi-of-strategic-it-cost-management
- Buttondown Preview (Markdown or Fancy Mode does not change result: 

<img width="600" alt="Screenshot 2025-04-13 at 1 06 33 PM" src="https://github.com/user-attachments/assets/885ed3cf-aa93-44da-86b6-e4f3200e9f97" />

**Reply by jmduke** [reply]
cc @mary-ext who's also looking into this. definitely agree that these should be standardized, and Jay I know we've had some issues here in the past that were due to ~bugs~ eccentricities in how LinkedIn handles OG tags, but I think this is a different thing

**Reply by mary-ext** [reply]
Looking into capping it into a set height at the moment, but I definitely think it's reasonable to have some way of making it even smaller if needed (a square image to the left instead of above)
